README.md

```diff
@@ -20,9 +20,12 @@ Run the extension container with:
 ```bash
 docker run --rm -p3000:80 sawjan/web-app-3dmodel-viewer
 ```
+You can access the app at https://host.docker.internal:9200
```

Comment:
this would be wrong here because this section doesn't say about serving ocis.
What can be done here is, remove `docker run` example and only say which port does the image expose. like this:
```diff
- Run the extension container with:
- ```bash
- docker run --rm -p3000:80 sawjan/web-app-3dmodel-viewer
- ```
+ Exposed port: `80`
```

Reply:
I like that idea. I think that makes it clearer. And I guess for those who use Docker frequently, I guess the would see from the command that it is exposed at port 80